src/viewProviders/baseModels/multiViewBase.ts

```diff
@@ -15,7 +15,7 @@ export abstract class ViewProviderDelegate<
 
   abstract children: T[];
 
-  abstract fetchChildren(resource: P): Promise<T[]>;
+  abstract fetchChildren(resource: P, forceDeepRefresh: boolean): Promise<T[]>;
```

Comment:
Same question here about setting a default:
```suggestion
  abstract fetchChildren(resource: P, forceDeepRefresh: boolean = false): Promise<T[]>;
```

Reply:
Same answer -- to force caller to declare what behavior they demand.